### ISSUE TITLE
ReloadData() is missing

### ISSUE THREAD

**Issue by kevinkub** [Mozharovsky/CVCalendar#47]
Hey, 

thank you for writing this awesome control. Currently I am designing a calendar which updates in the background. However I can not be sure to have all the data available, as soon as the view did appear. I am searching for a possibility to reload the data (for the dots and dots color) of the calendar view. I tried a few solutions like 
```
let monthView = self.calendarView.contentController.presentedMonthView
monthView.reloadWeekViewsWithMonthFrame(monthView.frame)
self.calendarView.commitCalendarViewUpdate()
self.calendarView.toggleTodayMonthView()
```
but nothing seems to work in a nice manner.

I hope you can help me or add support for this in the next version.

Thank you in advance.

**Comment by Mozharovsky** [comment]
Hello @kevinkub, 

Thank you for using this software. 

CVCalendar is going to have more features with an upcoming release. One you actually need is included. You'll be able to set multiple dots and update them once it is necessary. You can find the change log here #28. 

Keep watching, more cool things to come! 

Regards, 
Eugene

**Comment by patricks** [comment]
Hi,

I the same problem, is there any way to reload the calendar view, after it is loaded?

I also get dates in the background. If I swipe 3 month forward and swipe back after that, I can see the new events.

**Comment by kevinkub** [comment]
I ended up using https://github.com/jonathantribouharet/JTCalendar in the next version of my app. However you should check out the beta branch of this repo, although I have not had a look myself.

**Comment by matlamenace** [comment]
Hello,

 I'm having the same problem. Reloading the dotMarkers on the fly accordingly to my data updates.
 
 Any luck ?

 Because I've been struggling for a quite a while now...

**Comment by Mirant123** [comment]
Is there any method when i can set multiple dots? If is there please let me know only that method.

**Comment by jose920405** [comment]
https://github.com/Mozharovsky/CVCalendar/issues/145

**Comment by elsesiy** [comment]
Has been merged with #155 and thus closed. If there's more to it let me know, I'm gonna reopen the issue then..
